tests/GraphTest.php

```diff
@@ -44,18 +44,17 @@ public function testRequestWithCustomEndpoint()
         $graph->setBaseUrl('url2');
 
         $request = $graph->createRequest("GET", "/me");
-        $requestUrl = $this->readAttribute($request, 'baseUrl');
-        $this->assertEquals('url2', $requestUrl);
+        $this->assertEquals('url2', $request->getBaseUrl());
     }
 
     public function testBetaRequest()
     {
         $graph = new Graph();
         $graph->setAccessToken('abc')
-              ->setApiVersion('/beta');
+              ->setApiVersion('beta');
         $request = $graph->createRequest("GET", "/me");
 
-        $this->assertEquals('/beta', $this->readAttribute($request, 'apiVersion'));
+        $this->assertEquals('beta', $request->getApiVersion());
```

Comment:
Could you please confirm if not passing in `/` will create the correct URL using the HttpClient?

Reply:
To piggyback on this [comment ](https://github.com/microsoftgraph/msgraph-sdk-php/pull/418/files#r594530989)as a related note, we should get some clarity on how we handle leading `/` throughout the SDK. Ideally, the customer doesn't have to add it.

Reply:
I can confirm that it works without the leading `/`. Guzzle handles both cases (with or without the leading `/`) to generate the correct URL based on some [relative  URI resolution standard](https://tools.ietf.org/html/rfc3986#section-5.2). The [Guzzle `base_uri` docs](https://docs.guzzlephp.org/en/stable/quickstart.html) provide a good summary table

Reply:
That's awesome. Thanks for the confirmation!